source/containers.tex

```diff
@@ -9200,6 +9238,665 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec2[inplace.vector]{Class template \tcode{inplace_vector}}
+
+\rSec3[inplace.vector.overview]{Overview}
+
+\pnum
+\indexlibraryglobal{inplace_vector}%
+An \tcode{inplace_vector} is a contiguous container.
+Its capacity is fixed and
+its elements are stored within the \tcode{inplace_vector} object itself.
+
+\pnum
+An \tcode{inplace_vector} meets all of the requirements
+of a container\iref{container.reqmts},
+of a reversible container\iref{container.rev.reqmts},
+of a contiguous container, and
+of a sequence container,
+including most of the optional sequence container requirements\iref{sequence.reqmts}.
+The exceptions are the
+\tcode{push_front},
+\tcode{prepend_range},
+\tcode{pop_front}, and
+\tcode{emplace_front}
+member functions, which are not provided.
+Descriptions are provided here only
+for operations on \tcode{inplace_vector} that
+are not described in one of these tables or
+for operations where there is additional semantic information.
+
+\pnum
+For any \tcode{N},
+\tcode{inplace_vector<T, N>::iterator} and
+\tcode{inplace_vector<T, N>::const_iterator}
+meet the constexpr iterator requirements.
+
+\pnum
+%FIXME: Should this instead say something like:
+%FIXME:     "For any \tcode{N} such that \tcode{N > 0} is \tcode{true}, ..."?
+%FIXME: Same for similar expressions added in P0843R14?
+For any \tcode{N > 0},
+if \tcode{is_trivial_v<T>} is \tcode{false}, then
+no \tcode{inplace_vector<T, N>} member functions
+are usable in constant expressions.
+
+\pnum
+Any member function of \tcode{inplace_vector<T, N>} that
+would cause the size to exceed \tcode{N}
+throws an exception of type \tcode{bad_alloc}.
+
+\pnum
+Let \tcode{IV} denote a specialization of \tcode{inplace_vector<T, N>}.
+If \tcode{N} is zero, then
+\tcode{IV} is both trivial and empty.
+Otherwise:
+\begin{itemize}
+\item
+If \tcode{is_trivially_copy_constructible_v<T>} is \tcode{true}, then
+\tcode{IV} has a trivial copy constructor.
+\item
+If \tcode{is_trivially_move_constructible_v<T>} is \tcode{true}, then
+\tcode{IV} has a trivial move constructor.
+\item
+If \tcode{is_trivially_destructible_v<T>} is \tcode{true}, then:
+  \begin{itemize}
+  \item
+  \tcode{IV} has a trivial destructor,
+  \item
+  if
+\begin{codeblock}
+  is_trivially_copy_constructible_v<T> && is_trivially_copy_assignable_v<T>
+\end{codeblock}
+  is \tcode{true}, then
+  \tcode{IV} has a trivial copy assignment operator,
```

Comment:
```suggestion
  \tcode{IV} has a trivial copy assignment operator, and
```
(Assuming I've parsed the English correctly: this `itemize` is a list of three things, and the penultimate thing should end with `, and`. Compare other places that we do lists of things.)

Reply:
Thanks - fixed by making the sub-list statements as well.